### ISSUE TITLE
New provider version have incompatibilities with module

### ISSUE THREAD

**Issue by ToROxI** [lean-delivery/tf-module-aws-gitlab-runner#20]
Terraform v0.11.13
+ provider.aws v2.3.0

```
Error: module.gitlab-runner.data.aws_region.current: "current": [REMOVED] Defaults to current provider region if no other filtering is enabled
```

**Comment by ToROxI** [comment]
Also data source aws_ami has been changed.

Error: module.gitlab-runner.data.aws_ami.amazon_optimized_amis: "owners": required field is not set

**Comment by ToROxI** [comment]
Last compatible provider version - 1.60.0
```
Warning: module.gitlab-runner.data.aws_region.current: "current": [DEPRECATED] Defaults to current provider region if no other filtering is enabled
```

**Comment by lab-lamz4** [comment]
Fixed by #21
